R/sort_hierarchical.R

```diff
@@ -216,36 +216,41 @@ sort_hierarchical.tbl_hierarchical <- function(x, sort = everything() ~ "descend
   # add dummy rows for variables not in include so their label rows are sorted correctly
   not_incl <- setdiff(ard_args$variables, ard_args$include)
   if (length(not_incl) > 0) {
-    cli::cli_inform(
-      "Not all hierarchy variables present in the table were included in the {.arg include} argument.
-      These variables ({not_incl}) do not have event rate data available so the total sum of the event rates
-      for this hierarchy section will be used instead. To use true event rates for all sections of the table,
-      set {.code include = everything()} when creating your table."
-    )
+    if ("hierarchical" %in% x$context) { # do not print message for hierarchical count tables
```

Comment:
Instead of `"hierarchical" %in% x$context`, can we use `names(tbl$call_list)`? The context column is a bit weird in some cases, and its future is unknown, while the call list is native to gtsummary.

Reply:
Updated.